addons/mail/static/src/views/fields/mail_server_configurator_selection/mail_server_configurator_selection.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <div t-name="mail.MailServerConfiguratorSelection" class="d-flex flex-row">
```

Comment:
Extending a field but providing an entirely different template that does not extend the base field template seems like a bad idea.

Means it will fail if they change the implementation of "SelectionField".

Goes well with my earlier suggestion to rework this as a custom Component.

Reply:
but anyway, with a component it will need one more RPC, just for that it's not worth no ? (or adding the field in invisible in the view)

> Here you're using "string", "options", assuming that "options" is based on a tuple of 2 elements. You're using "value", etc etc.

but so they will break a lot of overwrite anyway no ?

I feel like doing the custom component and overwrite in sub-module is over-engineered and not the "odoo way". We will duplicate the options in python and in js (it will still need to be computed on the server side)